### PR TITLE
LA-7452 Add permission script for bigquery

### PR DESCRIPTION
**Pull request by kaif-lb**

### CODE REVIEW

File: sql_user_check_bigquery/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+### Pre-requisites
+
+Install [bq](https://cloud.google.com/sdk/docs/install-sdk) on the machine.
+
+Instruction to get service account json file [service account](https://docs.google.com/document/d/158OOKwQxv8bxotKwkuy5FLZ3or-0obYMwj0FKXUyrTw/edit#heading=h.hiriaxi7wof2)
+
+Authenticate to bigquery 
+
+``
+gcloud auth activate-service-account  --key-file=<service-account.json file path> --project=<project id>
```
Comment (arpit-lb) [inline sql_user_check_bigquery/README.md:11]: Add in readme what's the project ID and how a user is supposed to acquire this value

**Review by arpit-lb — CHANGES_REQUESTED** [review]
Please put correct ticket ID in the title.

File: sql_user_check_bigquery/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+for easier management e.g. dev, prod. It is comparable to an AWS sub account or AWS account. It can be obtained from 
+top of google cloud console page. It looks something like this `essential-smoke-393811`.
+
+
+``
+gcloud auth activate-service-account  --key-file=<service-account.json file path> --project=<project id>
+``
+
+### Run the script (from sql_user_check_bigquery directory)
+
```
Comment (arpit-lb) [inline sql_user_check_bigquery/README.md:18]:
```suggestion
### Run the script

Run the following command from `sql_user_check_bigquery` directory

```

**Review by arpit-lb — CHANGES_REQUESTED** [review]

File: sql_user_check_bigquery/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+stats Mode:
+```shell
+MODE=stats DATASETID=<dataset id> OUTPUT=<output file path>  bash run.sh
```
[the remark this answers is not in the document]
Comment (kaif-lb) [reply sql_user_check_bigquery/README.md]: added support for:
- multiple datasetIDs
- 0 datasetID

**Review by arpit-lb — APPROVED** [review]

**Pull request merged**